packages/main/src/Breadcrumbs.js

```diff
@@ -401,9 +405,10 @@ class Breadcrumbs extends UI5Element {
 			items = this.getSlottedNodes("items"),
 			item = items.find(x => `${x._id}-li` === listItem.id);
 
-		window.open(item.href, item.target || "_self", "noopener,noreferrer");
 		this.responsivePopover.close();
-		this.fireEvent("item-click", { item });
+		if (this.fireEvent("item-click", { item }, true)) {
```

Comment:
Hi @nnaydenow,
if I get it right, you suggest to keep the popover open, if preventDefault is called, correct?
Problem with this solution is that in our usecase this is not the expected behaviour. We would then need a way to "manually" close the popover, since we only react on item-click and need the href only for opening in new tab.

Reply:
Could you please provide information about your use case so we can discuss it with the team?

Reply:
Hi @nnaydenow,
after checking your proposed solution with our scenario, it turned out, that it seems to be not a problem, because we remove all breadcrumb items and add new ones on route change. The removal of all items causes also the popup to be removed, so this is our desired state.
I have pushed your suggestion.

While checking, I stumbled upon another issue, so stay tuned for another PR ;) 

Since our project is internal, I have created a small gist to mimic it, if you are interested, you can check it out here: https://gist.github.com/hardl/96979b1425f3de62e52cea73042d9c31